parse/dep_test.go

```diff
@@ -0,0 +1,56 @@
+package parse
+
+import (
+	"testing"
+)
+
+func TestReadGopkg(t *testing.T) {
+	//filepath := "../Gopkg.lock"
```

Comment:
Is this meant to be fully commented out?

Reply:
The reason that this one is commented out is that it relies on the existence of a gopkg file in the repo, which does not exist since I implemented go modules. I will file an issue to create a test gopkg file that can be used instead.